recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.9.0" %}
+{% set version = "3.11.0" %}
 # if build_num is reset to 0 (for new version), update increment for blas_minor below
-{% set build_num = 22 %}
+{% set build_num = 0 %}
 {% set version_major = version.split(".")[0] %}
 # blas_major denotes major infrastructural change to how blas is managed
 {% set blas_major = "2" %}
 # make sure we do not create colliding version strings of output "blas"
 # for builds across lapack-versions within the same blas_major
-{% set blas_minor = build_num + 100 %}
+{% set blas_minor = build_num + 300 %}
```

Comment:
I'm leaving this as is so we keep the `+ 200` for LAPACK 3.10, if we ever need to build that for whatever reason.